build/cloud/ha/03-ha-deployment.yaml

```diff
@@ -18,6 +18,8 @@ spec:
         k8s-app: kubeedge
         kubeedge: cloudcore
     spec:
+      nodeSelector: # configure the nodeSelector here to directly schedule pods to specific nodes
+        [key]: [value]
```

Comment:
If this file is not a turnkey yaml, probably name it as `03-ha-deployment.yaml.example` instead?

Reply:
> If this file is not a turnkey yaml, probably name it as `03-ha-deployment.yaml.example` instead?

I get it.